gnomock_test.go

```diff
@@ -192,6 +192,42 @@ func TestGnomock_withCommand(t *testing.T) {
 	require.NoError(t, r.Close())
 }
 
+func TestGnomock_witUseLocalImagesFirst(t *testing.T) {
+	t.Parallel()
+
+	const (
+		mongoImage         = "docker.io/library/mongo:4.4"
+		circleciMongoImage = "docker.io/circleci/mongo:4.4"
+	)
+
+	container, err := gnomock.StartCustom(
+		mongoImage,
+		gnomock.DefaultTCP(testutil.GoodPort80),
+		gnomock.WithUseLocalImagesFirst(),
+	)
+	require.Nil(t, err)
```

Comment:
The result would be the same, but I personally prefer to use `require.NoError` as it producer more clear test failure output ("unexpected error").